parts/dcosmasterresources.t

```diff
@@ -141,9 +141,8 @@
         "name": "nicLoopNode"
       },
       "dependsOn": [
-{{if .MasterProfile.IsCustomVNET}}
         "[variables('masterNSGID')]",
-{{else}}
+{{if not .MasterProfile.IsCustomVNET}}        
```

Comment:
Right. In my templates I normally have `[resource('RP/', variable('resourceName')]` which is the exact thing you're doing. Don't know why I was confused. Thanks.